lib/pusher/push_notifications/client.rb

```diff
@@ -30,7 +28,24 @@ def post(resource, payload = {})
             body = JSON.parse(response.body)
             Response.new(status, body, status == 200)
           else
-            Response.new(500, nil, false)
+            Response.new(status, nil, false)
+          end
+        end
+      end
+
+      def delete(resource)
```

Comment:
It's odd to call it `resource` when you already know that 100% of the cases it's a `user`.

Reply:
Fixed in 0177cc5.